### PR TITLE
update to Angular 14

### PR DESCRIPTION
**Pull request by pburgmer**
* supports only Angular 14
* intermediate update steps have not been tested

### CODE REVIEW

**Review by alberthaff — COMMENTED** [review]
Hi.
Thank you for updating the library to support the latest Angular version!

It looks good - I just have a question regarding the change of the dependencies.

File: package.json [hunk 1]
```diff
@@ -44,49 +44,50 @@
...
     "start": "ng serve",
-    "build": "ng build ngx-papaparse --prod",
+    "build": "ng build ngx-papaparse --configuration production",
     "test": "ng test ngx-papaparse --browsers PhantomJS --watch=false",
     "test:coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "lint": "ng lint",
     "e2e": "ng e2e"
   },
   "dependencies": {
-    "papaparse": "^5.3.2",
```
Comment (alberthaff) [inline package.json:54]: Won't this require the user to manually install papaparse?

Comment (pburgmer) [reply package.json:54]: It is a peer dependency now and those are installed automatically with newer npm versions. No manual interation required, same as before. Advantage is that we avoid installation of multiple different versions in sub folders.

Comment (alberthaff) [reply package.json:54]: Ah, i was not aware they made that change to NPM. 👍

**Pull request merged**